packages/gestalt/src/Tile/Tile.js

```diff
@@ -0,0 +1,161 @@
+// @flow strict
+import { type Node, useEffect, useState } from 'react';
+import classnames from 'classnames';
+import Checkbox from '../Checkbox.js';
+import Flex from '../Flex.js';
+import { ENTER } from '../keyCodes.js';
+import Tooltip from '../Tooltip.js';
+import useFocusVisible from '../useFocusVisible.js';
+import useInteractiveStates from '../utils/useInteractiveStates.js';
+import { FixedZIndex } from '../zIndex.js';
+import styles from './Tile.css';
+
+type Props = {|
```

Comment:
Is this component exposed publicly? If not, there's no need for these prop descriptions except where they provide value to Future Us

Reply:
This conversation was marked as "resolved" but no changes were made, and there was no explanation. Please do not do that.

This question remains: are we exposing this component publicly? If not, no need for these comments.

Reply:
I'm still seeing the prop description comments. If there is a difference between this component's props and the props of external components, we should call that out. Otherwise these comments are a second source of truth and there will likely be drift with the information surfaced in the docs.